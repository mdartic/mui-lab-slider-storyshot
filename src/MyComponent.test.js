import initStoryshots from '@storybook/addon-storyshots'

initStoryshots({
  config: ({ configure }) =>
    configure(() => {
      require('./MyComponent.stories.js')
    }, module)
})
