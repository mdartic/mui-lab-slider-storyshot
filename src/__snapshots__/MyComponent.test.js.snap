// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots MyComponent default 1`] = `
<div
  aria-labelledby="label"
  aria-orientation="horizontal"
  aria-valuemax={100}
  aria-valuemin={0}
  aria-valuenow={50}
  className="MuiSlider-root-1"
  onClick={[Function]}
  onMouseDown={[Function]}
  onTouchMove={[Function]}
  onTouchStartCapture={[Function]}
  role="slider"
>
  <div
    className="MuiSlider-container-2"
  >
    <div
      className="MuiSlider-track-3 MuiSlider-trackBefore-4"
      style={
        Object {
          "width": "50%",
        }
      }
    />
    <button
      className="MuiButtonBase-root-14 MuiSlider-thumb-6"
      onBlur={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      onTouchStartCapture={[Function]}
      style={
        Object {
          "left": "50%",
        }
      }
      tabIndex="0"
      type="button"
    />
    <div
      className="MuiSlider-track-3 MuiSlider-trackAfter-5"
      style={
        Object {
          "width": "calc(100% - 5px)",
        }
      }
    />
  </div>
</div>
`;

exports[`Storyshots MyComponent fixed 1`] = `
<div
  aria-labelledby="label"
  aria-orientation="horizontal"
  aria-valuemax={100}
  aria-valuemin={0}
  aria-valuenow={50}
  className="MuiSlider-root-17"
  onClick={[Function]}
  onMouseDown={[Function]}
  onTouchMove={[Function]}
  onTouchStartCapture={[Function]}
  role="slider"
>
  <div
    className="MuiSlider-container-18"
  >
    <div
      className="MuiSlider-track-19 MuiSlider-trackBefore-20"
      style={
        Object {
          "width": "50%",
        }
      }
    />
    <button
      className="MuiButtonBase-root-30 MuiSlider-thumb-22"
      onBlur={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      onTouchStartCapture={[Function]}
      style={
        Object {
          "left": "50%",
        }
      }
      tabIndex="0"
      type="button"
    />
    <div
      className="MuiSlider-track-19 MuiSlider-trackAfter-21"
      style={
        Object {
          "width": "calc(100% - 5px)",
        }
      }
    />
  </div>
</div>
`;
